test/test-shutdown.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/python3
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# Copyright (c) 2021 SUSE LLC
+#
+
+import argparse
+import logging
+import pexpect
+import sys
+
+
```

Comment:
doubled blank line.

Reply:
python coding style likes that though. flake8 complains :)

Reply:
OK.